docs/notebook_requirements.txt

```diff
@@ -18,7 +18,7 @@ jupyter-console==5.2.0
 jupyter-core==4.4.0
 kiwisolver==1.0.1
 MarkupSafe==1.0
-matplotlib==2.2.2
+matplotlib==3.0.3
```

Comment:
This seems to break python 2.7.  Any reason that this is required?

Reply:
I'll change it back to 2.2.2. We should abandon python 2 soon anyway, it is officially deprecated at the beginning of 2020.

Reply:
Thanks.  Is there a specific advantage of 3.0.3 over 2.2.2, just so we can be aware for future use?  I would be ok with ditching python 2.7 support early if we do a major RdTools revision that requires it, but if it's not too much hassle to keep 2.7 going, then I figure why not...

Reply:
No specific advantage that I know of, it's just what I had in the environment I used to develop this :-)